walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -13,11 +13,111 @@
 // limitations under the License.
 
 package com.google.sps;
-
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.Set;
+import java.util.HashSet;
 
 public final class FindMeetingQuery {
+  private static final int TIME_INCREMENT = 1;
+
+  /**
+   * Return a collection of TimeRange objects representing the available times when
+   * all attendees from request are available to meet.
+   * events is a collect of Events of when at least one attendee is busy. The Events
+   * in events can be considered busy times when the meeting can't be scheduled.
+   */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
```

Comment:
A TODO usually goes inside a comment instead of the exception. The exception should just have something along the lines of "This operation is unsupported."